### PR TITLE
Footer component PR

### PR DESCRIPTION
**Pull request by wchrapcynski**
Footer component is complete for issue #18. Some of the CSS will need to be adjusted once the variables are merged in for use. Also, the links in the footer will just been hard links. Those will need to be updated as information becomes available.

### CODE REVIEW

**Comment by ZakkMan** [comment]
@wchrapcynski Make sure you reference the issue in your PRs

File: client/src/components/Footer/Footer.css [hunk 1]
```diff
@@ -0,0 +1,47 @@
+@import url('https://fonts.googleapis.com/css?family=Reem+Kufi&display=swap'); /* Will eventually delete */
```
Comment (ZakkMan) [inline client/src/components/Footer/Footer.css]: This should be set globally, in `App.css`

File: client/src/components/Footer/Footer.css [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+.Footer {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    min-width: 100vw;
+    height: 50px;
+    background-color: #FFCB06; /* Change colors once css variables are merged */
+    font-family: 'Reem Kufi', sans-serif; /* Will eventually delete */
```
Comment (ZakkMan) [inline client/src/components/Footer/Footer.css]: The variables are merged, so you should be good to use those!

File: client/src/components/Footer/Footer.css [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+    .Footer {
+        height: 75px;
+    }
+
+    .footer-right {
+        flex-direction: column;
+        justify-content: right;
+        line-height: 20px;
+    }
+}
```
Comment (ZakkMan) [inline client/src/components/Footer/Footer.css]: This is a great start to responsive! On mobile can we have all three columns stack?

Comment (wchrapcynski) [reply client/src/components/Footer/Footer.css]: @ZakkMan Made edits and pushed them up. The footer collapses to a single column below 400px. All should be ready.

File: client/src/components/Footer/Footer.css (hunk 1 above)
Comment (wchrapcynski) [reply client/src/components/Footer/Footer.css]: Updated the component CSS to reflect this.

**Pull request merged**